docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-alerts.md

```diff
@@ -86,20 +103,34 @@ Consider increasing the disk space per node. By default, you are entitled to 50G
 
-- Node CPU use exceeds 70% on average for at least 5 minutes (Warning).
-- Node CPU use exceeds 90% on average for at least 5 minutes (Severe).
+- Up to 34% of all nodes in the cluster are reporting as down (Info).
+- More than 34% of all nodes in the cluster are reporting as down (Warning).
+- More than 66% of all nodes in the cluster are reporting as down (Severe).
 
-If your cluster experiences frequent spikes in CPU use, consider optimizing your workload.
+If fewer than 34% of nodes in a multi-node (that is, highly available) cluster are down, the cluster remains healthy and can continue to serve requests.
 
-Unoptimized queries can lead to CPU alerts. Use the [Slow Queries](../cloud-queries-slow/) and [Live Queries](../cloud-queries-live/) views to identify potentially problematic queries, then use the EXPLAIN statement to see the query execution plan and identify optimizations. Consider adding one or more indexes to improve query performance. For more information, refer to [Analyzing Queries with EXPLAIN](../../../explore/query-1-performance/explain-analyze/).
+If more than 66% of nodes in a multi-node (that is, highly available) cluster are down, the cluster is considered unhealthy and the downed nodes should be replaced as soon as possible.
 
-High CPU use could also indicate a problem and may require debugging by {{% support-cloud %}}.
+When cluster nodes go down, Yugabyte is notified automatically and will restore the nodes as quickly as possible.
 
-If CPU use is continuously higher than 80%, your workload may also exceed the capacity of your cluster. Consider scaling your cluster by adding vCPUs. Refer to [Scale and configure clusters](../../cloud-clusters/configure-clusters/).
+#### Fix memory alerts
+
+YugabyteDB Managed sends a notification when memory use in the cluster exceeds the threshold, as follows:
+
+- Memory use exceeds 75% (Warning).
+- Memory use exceeds 90% (Severe).
+
+If your cluster experiences frequent spikes in memory use, consider optimizing your workload.
+
+Unoptimized queries can lead to memory alerts. Use the [Slow Queries](../cloud-queries-slow/) and [Live Queries](../cloud-queries-live/) views to identify potentially problematic queries, then use the `EXPLAIN` statement to see the query execution plan and identify optimizations. Consider adding one or more indexes to improve query performance. For more information, refer to [Analyzing Queries with EXPLAIN](../../../explore/query-1-performance/explain-analyze/).
+
+If memory use is continuously higher than 80%, your workload may also exceed the capacity of your cluster. If the issue isn't a single query that consumes a lot of memory on a single tablet, consider scaling your cluster by adding nodes to lower the average per-node workload. Adding vCPUs also provides additional memory. Refer to [Scale and configure clusters](../../cloud-clusters/configure-clusters/).
```

Comment:
`consider scaling your cluster by adding nodes to lower the average per-node workload. Adding vCPUs also provides additional memory.` is this how we want to phrase it? Adding more CPUs doesn't technically increase memory, it's just that the two are correlated due to the instance types we use under the hood. Could we say something like `consider scaling your cluster either vertically (to increase per-node capacity) or horizontally (reduce per-node load).`?

Reply:
We should also have similar wording in the CPU alerts section.